launch/tier4_planning_launch/config/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/obstacle_avoidance_planner.param.yaml

```
@@ -19,6 +19,8 @@
       skip_optimization: false # skip MPT and EB
       reset_prev_optimization: false
 
+      enable_boost_check: true # enable boost check
```

It may be more user friendly if we add more description in the comment.
```suggestion
      enable_boost_check: true # use boost implementation for polygon intersection check
```

---

I added it.